testcases/kernel/syscalls/fallocate/fallocate01.c

```diff
@@ -84,6 +84,17 @@
  *		Cleanup the temporary folder
  *
 *************************************************************************/
+/*
+ * Patch Description: 
+ * Test Failure reason in SGX-LKL: 
+ * [[  SGX-LKL ]] libc_start_main_stage2(): Calling app main: /ltp/testcases/kernel/syscalls/fallocate/fallocate01
+ * fallocate01    1  TCONF  :  fallocate01.c:233: fallocate system call is not implemented
+ * Test is happening on temporary file created under /tmp, on which fallocate system call is failing.
+ *
+ * Workaround to fix the issue:
+ * fallocate system call is failing on temporary file created under /tmp directory.
+ * So modified the tests to create directory in root filesystem.  
+ */
```

Comment:
Change to:

* Patch Description: 
  Test Failure reason in SGX-LKL: 
  [[  SGX-LKL ]] libc_start_main_stage2(): Calling app main: /ltp/testcases/kernel/syscalls/fallocate/fallocate01
  fallocate01    1  TCONF  :  fallocate01.c:233: fallocate system call is not implemented
  Test is happening on temporary file created under /tmp, on which fallocate system call is failing.
 
 * Workaround to fix the issue:
  fallocate system call is failing on temporary file created under /tmp directory.
  So modified the tests to create directory in root filesystem.